src/components/Card.js

```diff
@@ -1,9 +1,27 @@
 export default class Card {
-  constructor(data, cardSelector, handleCardClick) {
+  constructor(
+    data,
+    cardSelector,
+    id,
+    handleCardClick,
+    handleCardDelete,
+    handleLike
+  ) {
     this._name = data.name;
     this._link = data.link;
     this._cardSelector = cardSelector;
     this._handleCardClick = handleCardClick;
+    this._owned = data.owner._id === id;
+    this._id = data._id;
+    this._handleCardDelete = handleCardDelete;
+    this._like = data.likes.length;
+    this._handleLike = handleLike;
+    this._likedByOwner = false;
+    data.likes.forEach((element) => {
```

Comment:
There is no need to have ` this._likedByOwner` variable, since the value can be calculated. Save the likes array and create a method `_isLiked`:

```js
_isLiked() {
 // return true if user liked the card, otherwise false
 this._likes.some((like) => like._id === this._userId)`
}
```